src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -37,7 +38,7 @@ public Driver create(Driver driver) {
 
     @Override
     public Optional<Driver> get(Long id) {
-        String query = "SELECT id, name, license_number "
+        String query = "SELECT driver_id, driver_name, driver_license_number "
```

Comment:
you don't need driver prefix for drivers table

Reply:
done